But why is it different between Bill's environment vs when running this under my environment?

Also in my case I see 
```
=========================== short test summary info ============================
FAILED src/esmpy/test/test_api/test_regrid.py::TestRegrid::test_field_regrid_zeroregion_select_ndbounds
====== 1 failed, 183 passed, 3 skipped, 10 xfailed, 7 warnings in 45.50s =======
dec2253.hsn.de.hpc.ucar.edu: rank 0 exited with code 1
```
At the end of https://github.com/esmf-org/esmf-test-artifacts/blob/b3a88c9f49cdfe90fe5b9bb59a0a5bb22e38ba0a/develop/gfortran/12.2.0_pio2.6.2/O/mpi/8.1.27/esmpy/esmpy9.0.0b0-petx1.test, which seems legit, I mean it is true that this test is failing.

--- a/python_scripts/collect_artifacts.py
+++ b/python_scripts/collect_artifacts.py
@@ -106,6 +106,7 @@ def _copy_test_artifacts():
     cmd.runcmd(f"mkdir -p {_out_dir}")
 
     _artifacts_list = glob.glob(f"{_test_dir}/esmf/src/addon/esmpy/*.test")
+    _artifacts_list.extend(glob.glob(f"{_test_dir}/esmf/src/addon/esmpy/*-summary.json"))
     _artifacts_list.extend(glob.glob(f"{_test_dir}/esmf/src/addon/esmpy/*.ESMF_LogFile"))
     for _f in _artifacts_list:
         _out = os.path.join(_out_dir, os.path.basename(_f))
@@ -207,14 +208,19 @@ def _create_summary():
         _esmpy_pass = 0
         _esmpy_fail = 0
         for _f in _esmpy_test_logs:
-            _esmpy_out = cmd.runcmd_no_err(f"tail -n 1 {_f}")
+            _jsonsf = re.sub(r"\.test$", "-summary.json", _f)
+            if not os.path.isfile(_jsonsf):
+                logging.debug(f"Summary file not found for esmpy test log {_f}")
+                _esmpy_out = ""
+            else:
+                _esmpy_out = cmd.runcmd_no_err(f"tail -n 1 {_jsonsf}")
             if '"summary":' in _esmpy_out:
                 _esmpy_pass_str = _extract(r'"passed": (\d+)', _esmpy_out, "0")
                 _esmpy_xfail_str = _extract(r'"xfailed": (\d+)', _esmpy_out, "0")
                 _esmpy_fail_str = _extract(r'"failed": (\d+)', _esmpy_out, "0")
                 _esmpy_xpass_str = _extract(r'"xpassed": (\d+)', _esmpy_out, "0")
             else:
-                logging.debug(f"Failed to find json summary at the end of {_f}")
+                logging.debug(f"Failed to find json summary at the end of {_jsonsf}")
                 _esmpy_pass_str = "0"
                 _esmpy_xfail_str = "0"
                 _esmpy_fail_str = "1"

